test/testcafe/spec/ProfileEnrollmentWithStringOptions_spec.js

```diff
@@ -50,4 +53,29 @@ test.requestHooks(requestLogger, ProfileEnrollmentSignUpWithStringFieldsMock)('s
   await t.expect(favPizza).eql('Razza');
 
   await t.expect(await profileEnrollmentString.getSaveButtonLabel()).eql('Sign Up');
+});
+
+test.requestHooks(requestLogger, ProfileEnrollmentSignUpWithStringFieldsMock)('should submit form when all optional fields are empty', async t => {
+  const profileEnrollmentString = new ProfileEnrollmentStringOptionsViewPageObject(t);
+  const identityPage = await setup(t);
+  await identityPage.clickSignUpLink();
+
+  await profileEnrollmentString.fillEmailField('test.carlos@mycompany.com');
+  await profileEnrollmentString.fillFirstNameField('Test Carlos');
+  await profileEnrollmentString.fillLastNameField('Test');
```

Comment:
can we add a field here with empty value ?

Reply:
I was re-checking my test and trying to see what else I was missing... but you found it!! you have a very good eye for details, thank you so much @pradeepdewda-okta